CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to `rising`
+
+If you are interested in contributing to `rising`, you will either
+
+* implement a new feature
+
+or 
+
+* fix a bug.
+
+For both types of contribution, the process is roughly the same:
+
+1. File an issue at [this repo] and discuss 
+the issue with us! Maybe we can give you some hints towards 
+implementation/fixing.
+
+2. If you're not part of the core development team, we need you to create your own fork of [this repo], implement it there and create a PR to [this repo] afterwards.
+
+3. Start a new branch (in your fork if necessary) for the implementation of your issue. 
+Make sure to include basic unittests.
+
+> **Note:** To improve readability and maintainability, [PEP8 Style](https://www.python.org/dev/peps/pep-0008/) should always be followed (no exceptions).
+
+> **Note:** We prefer absolute imports over realtive ones..
+
+> **Note:** If you added a feature, you should also add it to the documentation
+
+4. After finishing the coding part, send a pull request to the correct branch of [this repo] (probably master branch).
+
+5. Afterwards, have a look at your pull request since we might suggest some 
+changes.
+
+
+If you are not familiar with creating a Pull Request, here are some guides:
+- http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request
+- https://help.github.com/articles/creating-a-pull-request/
+
+
+## Development Install
+
+To develop `delira` on your machine, here are some tips:
+
+1. Uninstall all existing installs of `rising`:
+```
+pip uninstall rising
+pip uninstall rising # run this command twice
+```
+
+2. Clone a copy of `rising` from source:
+
+```
+git clone https://github.com/PhoenixDL/rising.git
+cd rising
+```
+
+3. Install `rising` in `build develop` mode:
+
+Install it via 
+
+```
+python setup.py build develop
+```
+
+or 
+
+```
+pip install -e .
+```
+
+This mode will symlink the python files from the current local source tree into the
+python install.
+
+Hence, if you modify a python file, you do not need to reinstall `rising` 
+again and again
+
+In case you want to reinstall, make sure that you uninstall `rising` first by running `pip uninstall rising`
+and `python setup.py clean`. Then you can install in `build develop` mode again.
+
+## Unit testing
+
+Unittests are located under `tests/`. Run the entire test suite with
```

Comment:
Maybe mention somewhere that these commands should be executed from the top directory `rising`